iogt_users/forms.py

```diff
@@ -32,10 +39,10 @@ class AccountSignupForm(SignupForm):
     def __init__(self, *args, **kwargs):
         super(AccountSignupForm, self).__init__(*args, **kwargs)
         self.fields.pop('email')
-        self.fields["password1"] = IogtPasswordField(label=_("4-digit PIN"), autocomplete="new-password")
+        self.fields["password1"] = IogtPasswordField(label=_("Enter a 4-digit PIN or a longer password"), autocomplete="new-password")
```

Comment:
I think this will have to be re-translated. I've asked whether this should be done now or as a separate issue.